apps/condo/domains/notification/tasks/deliverMessage.js

```diff
@@ -185,16 +187,27 @@ async function deliverMessage (messageId) {
             const isAllowedByUser = get(userTransportSettings, transport)
             if (isAllowedByUser === false) {
                 transportMeta.status = MESSAGE_DISABLED_BY_USER_STATUS
-                processingMeta.step = MESSAGE_SENT_STATUS
-                successCnt++
-            } else {
-                const [isOk, deliveryMetadata] = await _sendMessageByAdapter(transport, adapter, messageContext, isVoIP)
-                logger.info({ msg: 'sendMessageByAdapter', isOk, messageId, deliveryMetadata })
-                transportMeta.deliveryMetadata = deliveryMetadata
-                transportMeta.status = isOk ? MESSAGE_SENT_STATUS : MESSAGE_ERROR_STATUS
-                processingMeta.step = isOk ? MESSAGE_SENT_STATUS : MESSAGE_ERROR_STATUS
-                successCnt += isOk ? 1 : 0
+
+                processingMeta.step = MESSAGE_DISABLED_BY_USER_STATUS
```

Comment:
if the message disabled by one transport but enable by another?

Reply:
Fixed